styles/globals.css

```
@@ -19,3 +21,29 @@ main {
   padding: 4rem;
   flex-grow: 1;
 }
+
+.page-cover {
+  background-image: url(../assets/space.jpg);
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  padding-top: 90px;
+  width: 1920px;
```

Reemplazemos y probemos el valor de width
```suggestion
  width: 100%;
```